src/console.cpp

```diff
@@ -575,11 +575,12 @@ static void setup_commands(std::shared_ptr<Commands> & commands) {
                     return;
                 } else {
                     // show message if no data returned (e.g. for analogsensor, temperaturesensor, custom)
-                    shell.println("No data.");
+                    shell.println("No data returned.");
```

Comment:
This `no data` is confusing. It show for every command that have no json, 
E.g. For a `call system send ""` i don't expect a `no data` message, i only expect thet the command is executet.

Reply:
`No data` looks like a error message and is confusing. Maybe show in terminal a confirmation `command successful`, but i expect the command is executed if there is no error-message.

Reply:
done.